ERDAS_Imagine/kea.h

```diff
@@ -64,12 +64,6 @@
 // else in one of the earlier headers
 #define	DLL_INSTANCE_NAME	kea
 
-#ifdef _MSC_VER
-	// hack to get the descriptor table exported on Windows
-	// not sure if this affects all versions 
-	#define EDSC_API DllExport
-#endif
-
```

Comment:
This needs to be here before including `eimg_RasterFormats.h` for some reason. Can you just change `EDSC_API` to be `KEA_EXPORT`?

`DLL_INSTANCE_NAME` is also required before Imagine includes.

Reply:
Do you want `EDSC_API` to always be `__declspec(dllexport)` if msvc, or to be `KEA_EXPORT` (seems suspicious)?

Be aware that for consumers of libkea (libs or exec which link libkea), KEA_EXPORT will be:
-  `__declspec(dllimport)` if windows and libkea shared
- empty if windows and libkea static.

I suspect that you want `EDSC_API` to be always `__declspec(dllexport)` on Windows (or maybe just MSVC), and it's unrelated to `KEA_EXPORT`.

Reply:
Yes, `__declspec(dllexport)` is correct

Reply:
done ;)